dmoj/cptbox/_cptbox.pyx

```diff
@@ -445,6 +451,13 @@ cdef class Process:
         self._exited = True
         return self._exitcode
 
+    property use_seccomp:
+        def __get__(self):
+            return self.process.seccomp()
```

Comment:
We really call this thing too many things... so far there's

* `seccomp()`
* `use_seccomp`
* `avoid_seccomp`
* `_seccomp`